civo/terraform/outputs.tf

```
@@ -0,0 +1,54 @@
+output "civo_ssh_public_key" {
+  description = "The public key for SSH access to the Droplet instance"
```

```suggestion
  description = "The public key for SSH access to the Civo instance"
```
(Same for the other occurrences elsewhere in this file.)